### ISSUE TITLE
[Config Menu] Extend templates default hidden - not able to enable

### ISSUE THREAD

**Issue by matthys70** [Piwigo/Piwigo#1613]
I gone add this as bug report as it seems no follow-up from forum and not able to find in bug-list/issue (also not closed).

I notice I am missing the Extend templates under config menu, moving from 11.5.0 to 12.2.0 version. Suddenly this is by 'default' hidden. Why hide configurations by default? See also [Piwigo Forum](https://piwigo.org/forum/viewtopic.php?id=31850)

Should be solved with using the LocalFiles Editor and adding the following line in the Local config:
`$conf['show_template_in_side_menu'] = true;`

However, even if I use the default admin theme it's not working for me.
Currently running:
[Piwigo](https://piwigo.org/) 12.2.0 
Operating system: Linux (Ubuntu)
PHP: 7.4.16 [2022-02-21 11:15:55]
MySQL: 8.0.27-0ubuntu0.21.04.1 [2022-02-21 11:15:55]
Graphics Library: External ImageMagick 6.9.11-60

**Comment by plegall** [comment]
> Why hide configurations by default?

Because some features are advanced features and should not be "in the way" of the beginner. Concerning this possibility to extend templates, I would even say it's dangerous and useless. We keep it for legacy purpose.

**Comment by matthys70** [comment]
Does this means it is 'work in progress'?

Did not see the error messages but I'm NOT able the enable the hidden Extend templates in the config menu :-(

**Comment by matthys70** [comment]
I feel very embarrassed as it seems the problem is all my fault :-(

As I really didn't understand I did a diff from my Piwigo installation with the source.
And the diff shows that 4 files in the root where not the same, these files are:
admin.php, install.php, upgrade.php and ws.php

After I copied these files from source to my Piwigo installation, all problems where gone.

Now I wonder how did this happen and why didn't I notice before?
Well maybe because I always download the diff files, like the "12.x_to_12.2.0.zip".
But this package does contain the correct admin.php and upgrade.php (the others are not there).
The only other thing is that maybe not all files where copied do to some permission rights...

Anyway above solved these problems for me:
-After upload photo it's instantly in album (no delay)
-The Templates is under Configuration in the Admin (when enabled in the config)

Matthijs
